.github/workflows/release.yaml

```diff
@@ -9,8 +9,7 @@ on:
     paths-ignore:
       - 'tests/**'
       - 'docs/**'
-      - renovate.json
-      - poetry.lock
```

Comment:
We no longer have a separate charm requirements, so we should release if the lock is changed.